schemas/metadata/metaschema/metaschema.1.schema.json

```diff
@@ -57,12 +58,45 @@
                 "type": "string"
               }
             },
+            "required": [
```

Comment:
This is unrelated cleanup?

Reply:
Yes, I should have commented on this. While implementing `override_attributes`, I intentionally tried implementing some instances of the field incorrectly and found that the test suite was not picking up the errors, so I realized I need to disallow additionalProperties and specify required fields. I then went ahead and made those same improvements for the jwe_mappings schema.